### ISSUE TITLE
Guiding Path MTU will impact all IP traffic sharing path

### ISSUE THREAD

**Issue by gloinul** [quicwg/ops-drafts#325]
So Section 4.10 in -11 "Guiding Path MTU" discusses dropping QUIC packets above a configured limits that are less than the normal PMTU and also sending an ICMP PTB message back when doing this. I think the text in this section would benefit from an additional clarification that implementing this can impact all IP traffic to and from the involved endpoint pairs. The reasons for this is that RFC 8201 recommend that endpoints that receive IPv6 ICMPv6 PTBs cache these messages in relation to a implementation specific representation of the path. Thus, any IPv6 packets sent from the endpoint towards a destination which the implementation considers being the same path will have a PMTU applied according to the value in the PTB. 

For example if you have an endpoint in an access network and a datacenter front-end that fronts both HTTP/3 servers and a DNS service that the endpoint uses, then the DNS message size would be fragmented to the lower value than the absolute PMTU. 

I don't see any major issue with this, but I think the document should be clear on the potential implications of applying this method to QUIC flows.

**Comment by gorryfair** [comment]
Yes if a node updates it's IP cache (which is a valid approach), this impacts other flows sharing the same "path" where the notion of path needs thought (RFC8201, s5.2).

I'm noting that RFC8899 explicity says: "The PMTU value calculated from the PLPMTU MAY also be stored with the corresponding entry associated with the destination in the IP layer cache and used by other PL instances." This was a MAY because of the potential impact on other flows.

When the IPv6 RFC was updated, we added some caution about ECMP, etc. which is a hint at the same problem.

**Comment by gorryfair** [comment]
I can review this.
